src/pages/_app.tsx

```diff
@@ -26,23 +25,23 @@ const NAVIGATION: Navigation = [
     icon: <DashboardIcon />,
   },
   {
-    segment: 'orders',
-    title: 'Orders',
-    icon: <ShoppingCartIcon />,
+    segment: 'listings',
+    title: 'Your Listing',
```

Comment:
You need to hide this if. your user isn't logged in

Reply:
The your listing page also needs to be hidden if not logged in?